### PR TITLE
release/2.5.0

### PR DESCRIPTION
**Pull request by Zethson**
2.4.7 (2020-10-03)
------------------

**Added**

* verbose support #186

**Fixed**

* sync workflow now polls instead of being triggered on push #170
* renamed branch protection workflow #190
* refactored sync command

**Dependencies**

**Deprecated**

### CODE REVIEW

File: qube/qube_cli.py [hunk 1]
```diff
@@ -142,45 +161,42 @@ def sync(project_dir, set_token, pat, username, check_update) -> None:
     if set_token:
         try:
             project_data = load_yaml_file(f'{project_dir}/.qube.yml')
-            repo_owner = project_data['github_username']
-            # if project is an orga repo, pass orga name as username
-            if project_data['is_github_repo'] and project_data['is_github_orga']:
```
Comment (Zethson) [inline qube/qube_cli.py:147]: @sven1103 Why did we even pass the new repo_owner thingy here?
It was not used at all.

**Comment by Zethson** [comment]
@sven1103 
I made the sync code a little bit prettier and simpler.
Some of the code must origin from before I included the `creator_github_username` and the `github_username` distinction.

I would appreciate an in depth review of especially the sync changes. I don't want to mess it up now that it is running smoothly :)

Cheers.

(1 AM UTC = 3 AM our time -> sync will run at 3 AM from now on)

File: qube/qube_cli.py (hunk 1 above)
Comment (sven1103) [reply qube/qube_cli.py:147]: huh, yes indeed 🤦

**Review by sven1103 — CHANGES_REQUESTED** [review]
Adhering to semantic versioning, this is already a minor release as a new feature is added.

The new version should be `2.5.0` then.

In addition, I would like to get rid of a flattening pom plugin and introduce this change in the release as well.

**Comment by Zethson** [comment]
@sven1103 Yep, agreed.
Just wanted to fix one thing first, but then as always... :)

Bumped to 2.5.0 and improved the qube issue templates a little but to make them less verbose.

Could also add this to our templates.

**Comment by sven1103** [comment]
Dear @Zethson , thanks a lot!. 

> Bumped to 2.5.0 and improved the qube issue templates a little but to make them less verbose.
>
> Could also add this to our templates.

We already have an internal discussion, how these template should look like to meet our needs of detail vs. verbosity. We can leave it as it is for now until we came to a decision :)

Best. Sven

**Review by sven1103 — APPROVED** [review]
LGTM now, great 🎉

**Pull request merged**